### PR TITLE
Support multiple store URLs

### PR DESCRIPTION
**Pull request by jsravn**

### CODE REVIEW

**Comment by jsravn** [comment]
Depends on https://github.com/sky-uk/gorb/pull/7

File: core/store.go [hunk 1]
```diff
@@ -25,13 +25,29 @@ type Store struct {
...
+	var storePath string
+	var hosts []string
+	for _, storeURL := range storeURLs {
+		uri, err := url.Parse(storeURL)
+		if err != nil {
+			return nil, err
+		}
+		uriScheme := strings.ToLower(uri.Scheme)
+		if scheme != "" && scheme != uriScheme {
+			return nil, errors.New("schemas must be the same for all store URLs")
```
Comment (alangibson01) [inline core/store.go]: Typo: *schemes*

File: core/store.go [hunk 2]
```diff
@@ -41,21 +57,24 @@ func NewStore(storeUrl, storeServicePath, storeBackendPath string, syncTime int6
 	case "boltdb":
 		backend = store.BOLTDB
 	default:
-		return nil, errors.New("unsupported uri schema : " + uri.Scheme)
+		return nil, errors.New("unsupported uri schema : " + scheme)
```
Comment (alangibson01) [inline core/store.go]: Typo: *scheme*

File: main.go [hunk 3]
```diff
@@ -45,7 +46,8 @@ var (
 	listen           = flag.String("l", ":4672", "endpoint to listen for HTTP requests")
 	consul           = flag.String("c", "", "URL for Consul HTTP API")
 	vipInterface     = flag.String("vipi", "", "interface to add VIPs")
-	storeURL         = flag.String("store", "", "store url for sync data")
+	storeURLs        = flag.String("store", "", "comma delimited list of store urls for sync data. All urls must have" +
+		" identical schemas and paths.")
```
Comment (alangibson01) [inline main.go]: Typo: *schemes*

**Comment by jsravn** [comment]
@alangibson01 Hmm, I'm not sure about writing a store_test.go since it integrates directly with libkv. I can give it a shot.

**Comment by jsravn** [comment]
wip - found a way to mock out the store.

**Comment by jsravn** [comment]
@alangibson01 please have another look.

**Pull request merged**